src/main/java/jsr223/docker/compose/utils/DockerComposePropertyLoader.java

```diff
@@ -45,30 +46,25 @@
 @Log4j
 public class DockerComposePropertyLoader {
 
-    @lombok.Getter
-    public String dockerHost;
-
-    @lombok.Getter
-    public String dockerComposeCommand;
-
-    @lombok.Getter
-    public String sudoCommand;
-
-    @lombok.Getter
-    public boolean useSudo;
-
-    protected Properties properties;
-
-    protected String configurationFile = "config/scriptengines/docker-compose.properties";
-
-    private DockerComposePropertyLoader()  {
+    private final static String CONFIGURATION_FILE = "config/scriptengines/docker-compose.properties";
+    @Getter
+    private final String dockerHost;
+    @Getter
+    private final String dockerComposeCommand;
+    @Getter
+    private final String sudoCommand;
+    @Getter
```

Comment:
I don't want getter methods for all members. As an example:  a HashMap, if anyone is able to get a HashMap reference, then one can change the content of that HashMap from the outside.